src/dev-app/theme.scss

```diff
@@ -25,6 +25,7 @@ $candy-app-theme: mat.m2-define-light-theme((
 // have to load a single css file for Angular Material in your app.
 // **Be sure that you only ever include this mixin once!**
 @include mat.core();
+@include mat.elevation-classes();
```

Comment:
I think you can actually skip this - I dont think we use this in our dev-app. Maybe same in the universal app